api/src/log.rs

```diff
@@ -2,16 +2,22 @@
 // Parts of the project are originally copyright © Meta Platforms, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::metrics::{HISTOGRAM, RESPONSE_STATUS};
+use crate::metrics::{HISTOGRAM, REQUEST_SOURCE_CLIENT, RESPONSE_STATUS};
 use aptos_logger::{
     debug, info,
     prelude::{sample, SampleRate},
     warn, Schema,
 };
+use once_cell::sync::Lazy;
 use poem::{http::header, Endpoint, Request, Response, Result};
 use poem_openapi::OperationId;
+use regex::Regex;
 use std::time::Duration;
 
+const REQUEST_SOURCE_CLIENT_UNKNOWN: &str = "request-source-client-unknown";
```

Comment:
We could probably just shorten this to just `unknown`, it's contextual so we can save on some bytes.